docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -24,54 +24,50 @@
 
 @endsphinxdirective
 
-## Introduction
-
 Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
```

Comment:
```suggestion
Model Optimizer is a cross-platform command-line tool. It does not infer models but facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
```
A bit of rewording to avoid the disclaimer.

Reply:
I don't think we should start with stating what it does not do, like Maciek mentioned earlier. Following this approach, we should also change OV Runtime article to state it does not convert models. Let's leave it with "that."